bin/sign

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+ENV["PROJECT_ROOT"] = Dir.pwd
+
+# Begin warning: This sign script is an internal tool.
+# This is not intended to function outside of Puppet Labs' infrastructure.
+# End of warning.
```

Comment:
Needing to put this in a public repo makes me feel like something is wrong.
That being said, is this warning necessary? packaging is also a public repo.. Our keys and stuff don't live there, those are all build-data, so I'm not totally sure why this couldn't be used outside of Puppet?

Reply:
This is also copied from ship/repo. Packaging is public, but I think the methods definitely make some assumptions about infrastructure configuration that are non-obvious and specialized to our setup.

Someday it would be good to have these more specialized things not part of core vanagon, but that work shouldn't block this work.

Reply:
fair enough